get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
```

Comment:
I'm sure @jorgeorpinel will have strong opinions on titles & section titles to remain consistent with the current docs.

Reply:
_The DVC Get Started Project Focusing Mainly on Experiments (but may be used for other features too)_. 

We have two pages for the experiments in the docs, one is titled "Get Started: Experiments", and the other is "Experiment Management." "Get Started: Experiments" looks like a document title instead of a project name but I can use it here too @jorgeorpinel

Reply:
I think @casperdcl meant to keep just **DVC Get Started** here, as it is now (not sure).

Idk, is this repo now only intended to be used in the GS/Experiments and UG/Exp Mgmt pages? If so maybe **DVC Experiments Example** would be a better title.